.github/actions/spelling/excludes.txt

```diff
@@ -4,31 +4,40 @@
 (?:^|/)3rdparty/
 (?:^|/)go\.sum$
 (?:^|/)package(?:-lock|)\.json$
+(?:^|/)Pipfile$
 (?:^|/)pyproject.toml
 (?:^|/)requirements(?:-dev|-doc|-test|)\.txt$
 (?:^|/)vendor/
-ignore$
+(?:|$^ 90.22% - excluded 2693/2985)^gfx/
```

Comment:
The gfx/ directory does need to be scraped for spelling errors. Paradox Scripting Language, the language this repo is written in, uses formatting akin to CSS Flexbox but with nonstandard naming conventions, eg "flowcontainer" instead of "flow_contianer", which gets flagged as not being a word. However we do want to know if "flowcontainer" is spelled correctly.